pytest_pretty/__init__.py

```diff
@@ -98,3 +100,48 @@ def pytest_configure(config):
     custom_reporter = CustomTerminalReporter(config, sys.stdout)
     config.pluginmanager.unregister(standard_reporter)
     config.pluginmanager.register(custom_reporter, 'terminalreporter')
+
+
+ansi_escape = re.compile(r'(?:\x1B[@-_]|[\x80-\x9F])[0-?]*[ -/]*[@-~]')
+stat_re = re.compile(r'(\d+) (\w+)')
+
+
+def _new_outcomes(obj):
```

Comment:
Please can we:
* add type hints
* name the function something more descriptive, e.g. `create_new_parseoutcomes`
* name parameters something more descriptive than `obj` - e.g. `res_instance`
* add a docstring, if this is copied (as I assume), include a link to where it's copied from, also explain what's going on

Reply:
It is not copied, just inspired.